### PR TITLE
zh trans cloud-controller-manager.md

### PR DESCRIPTION
**Pull request by jiazxjason**
zh trans cloud-controller-manager.md

### CODE REVIEW

File: content/zh/docs/reference/command-line-tools-reference/cloud-controller-manager.md [hunk 1]
```diff
@@ -0,0 +1,500 @@
...
+weight: 28
+---
+
+{{% capture synopsis %}}
+
+<!--
+The Cloud controller manager is a daemon that embeds
+the cloud specific control loops shipped with Kubernetes.
+-->
+云控制管理器是一个守护进程，它嵌入了Kubernetes附带的云特定控制循环。
```
Comment (xiangpengzhao) [inline content/zh/docs/reference/command-line-tools-reference/cloud-controller-manager.md]: 中英文之间用空格隔开一下 :)

Comment (jiazxjason) [reply content/zh/docs/reference/command-line-tools-reference/cloud-controller-manager.md]: 已修改，请review

Comment (jiazxjason) [reply content/zh/docs/reference/command-line-tools-reference/cloud-controller-manager.md]: @xiangpengzhao

File: content/zh/docs/reference/command-line-tools-reference/cloud-controller-manager.md [hunk 2]
```diff
@@ -0,0 +1,500 @@
...
+    <col span="1" style="width: 10px;" />
+    <col span="1" />
+  </colgroup>
+  <tbody>
+
+    <tr>
+      <td colspan="2">--allocate-node-cidrs</td>
+    </tr>
+    <tr>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">Should CIDRs for Pods be allocated and set on the cloud provider.</td>
```
Comment (markthink) [inline content/zh/docs/reference/command-line-tools-reference/cloud-controller-manager.md]: 这些内容未翻译

**Comment by tanjunchen** [comment]
@jiazxjason  please update the file THX

**Comment by tengqm** [comment]
closing due to long time stale. pls feel free to reopen and improve.

**Pull request closed without merging**